src/app/courses/add-courses/courses-add.component.html

```diff
@@ -1,118 +1,114 @@
-      <planet-form-error-messages [control]="courseForm.controls.startDate"></planet-form-error-messages>
-    </div>
-    <div class="form-group col-4">
-      <label>End Date</label>
-      <input type="date" class="form-control" formControlName="endDate">
-      <planet-form-error-messages [control]="courseForm.controls.endDate"></planet-form-error-messages>
-    </div>
-    <div class="form-group radio col-4">
-      <div>Frequency</div>
-      <!-- radio buttons need same name to allow only one value to be selected! -->
-      <label>
-        <input type="radio" name="daysToggler" (change)="toogleWeekly(true)"> Daily
-      </label>
-      <label>
-        <input type="radio" name="daysToggler" (change)="toogleWeekly(false)"> Weekly
-      </label>
-    </div>
-  </div>
-  <div class="form-check form-check-inline" *ngIf="!showDaysCheckBox">
-    <div>Day</div>
-      <label class="form-check-label" *ngFor="let day of days">
-        <input class="form-check-input" type="checkbox" (change)="onDayChange(day, $event.target.checked)"> {{day}}
-      </label>
-  </div>
-  <div class="row">
-    <div class="form-group col-4">
-      <label>Start Time</label>
-      <input type="time" class="form-control" formControlName="startTime">
-      <planet-form-error-messages [control]="courseForm.controls.startTime"></planet-form-error-messages>
-    </div>
-    <div class="form-group col-4">
-      <label>End Time</label>
-      <input type="time" class="form-control" formControlName="endTime">
-      <planet-form-error-messages [control]="courseForm.controls.endTime"></planet-form-error-messages>
-    </div>
-    <div class="form-group col-4">
-      <label>Location</label>
-      <input type="text" class="form-control" formControlName="location">
-    </div>
-  </div>
-  <div class="row">
-    <div class="form-group col-2">
-      <label for="backgroundCol">Background Color</label>
-      <input type="color" class="form-control" formControlName="backgroundColor">
-      <planet-form-error-messages [control]="courseForm.controls.backgroundColor"></planet-form-error-messages>
-    </div>
-    <div class="form-group col-2">
-      <label>Foreground Color</label>
-      <input type="color" class="form-control" formControlName="foregroundColor">
-      <planet-form-error-messages [control]="courseForm.controls.foregroundColor"></planet-form-error-messages>
-    </div>
-  </div>
-  <button type="submit" class="btn btn-primary">Submit</button>
-  <button type="button" class="btn btn-danger" (click)="cancel()">Cancel</button>
-</form>
+<h1 i18n>Add Courses</h1>
+  <form class="addCourse-container" [formGroup]="courseForm" (ngSubmit)="onSubmit()" novalidate>
+    <mat-grid-list cols="3">
+      <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Course Title" formControlName="courseTitle" required>
+          <mat-hint><planet-form-error-messages [control]="courseForm.controls.courseTitle"></planet-form-error-messages></mat-hint>
```

Comment:
I think we need to use mat in form error messages rather than placing outside of form error messages.

Reply:
@razu9861 point @nirojdyola  mentioned is about `mat-error` which is the error component of material. We should be able to have it to use `planet-form-error-messages` feature.
Also `mat-hint` is supposed to be always shown hint that gives hint for input field not error

Reply:
@lmmrssa I think we might want to reconsider the `planet-form-error-messages` and use `mat-error` instead since it'll be simpler to implement for translation.